### PR TITLE
fix(ui): Fix Cannot read property 'includes' of undefined

### PR DESCRIPTION
**Pull request by priscilawebdev**
Fixes JAVASCRIPT-23F2

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugImage.tsx [hunk 1]
```diff
@@ -173,7 +173,9 @@ const DebugImage = React.memo(
       );
     };
 
-    const hasImagesLoadedV2Feature = organization.features.includes('images-loaded-v2');
+    const hasImagesLoadedV2Feature = !!organization?.features.includes(
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugImage.tsx:176]: I think the question mark should be after `features`

Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/events/interfaces/debugMeta/debugImage.tsx:176]:
```suggestion
    const hasImagesLoadedV2Feature = !!organization.features?.includes(
```

**Review by matejminar — APPROVED** [review]

**Comment by matejminar** [comment]
As a follow-up once this hot-fix lands we should improve typescript types (full organization might not be there because of shared view).

**Comment by priscilawebdev** [comment]
@matejminar good idea. I've created an asana task https://app.asana.com/0/1182975495223897/1199656936587421

**Pull request merged**